servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/SslConfig.java

```diff
@@ -186,4 +197,19 @@ default Duration handshakeTimeout() {   // FIXME 0.43 - remove default implement
     default int maxCertificateListBytes() {
         return 0;
     }
+
+    /**
+     * Defines filtering logic for {@link #ciphers() ciphers suites}.
+     */
+    enum CipherSuiteFilter {
+        /**
+         * Will take all requested ciphers suites as-is without any filtering.
+         */
+        IDENTITY,
```

Comment:
Consider using a different name here, I know `identity` is technically correct, but maybe something like `AS_PROVIDED` or `ALL_CONFIGURED` is more descriptive of which ones are used.

Reply:
(also since the user is not providing a function as an argument, related to the `identity function`)

Reply:
How about `PROVIDED` (with updated javadoc using similar wording) and `SUPPORTED`?
```
* Will take all {@link SslConfig#cipers() provided ciphers suites} as-is without any filtering.

Reply:
👍